sgv2-service-common/src/test/java/io/stargate/sgv2/common/cql/builder/QueryBuilderTest.java

```diff
@@ -0,0 +1,322 @@
+      arguments(
+          new QueryBuilder()
+              .alter()
+              .keyspace("ks")
+              .withReplication(Replication.simpleStrategy(1))
+              .build(),
+          "ALTER KEYSPACE ks "
+              + "WITH replication = { 'class': 'SimpleStrategy', 'replication_factor': 1 }"),
+      arguments(
+          new QueryBuilder()
+              .alter()
+              .keyspace("ks")
+              .withReplication(Replication.simpleStrategy(1))
+              .andDurableWrites(false)
+              .build(),
+          "ALTER KEYSPACE ks "
+              + "WITH replication = { 'class': 'SimpleStrategy', 'replication_factor': 1 } "
+              + "AND durable_writes = false"),
+      arguments(new QueryBuilder().drop().keyspace("ks").build(), "DROP KEYSPACE ks"),
+      arguments(
+          new QueryBuilder().drop().keyspace("ks").ifExists().build(),
+          "DROP KEYSPACE IF EXISTS ks"),
+      arguments(
+          new QueryBuilder()
+              .create()
+              .table("ks", "tbl")
+              .column("k", "int", Column.Kind.PARTITION_KEY)
+              .build(),
+          "CREATE TABLE ks.tbl (k int, PRIMARY KEY ((k)))"),
+      arguments(
+          new QueryBuilder()
+              .create()
+              .table("Ks", "Tbl")
+              .column("k", "int", Column.Kind.PARTITION_KEY)
+              .build(),
+          "CREATE TABLE \"Ks\".\"Tbl\" (k int, PRIMARY KEY ((k)))"),
+      arguments(
+          new QueryBuilder()
+              .create()
+              .table("tbl")
+              .column("k", "int", Column.Kind.PARTITION_KEY)
+              .build(),
+          "CREATE TABLE tbl (k int, PRIMARY KEY ((k)))"),
+      arguments(
+          new QueryBuilder()
+              .create()
+              .table("ks", "tbl")
+              .ifNotExists()
+              .column("k", "int", Column.Kind.PARTITION_KEY)
+              .build(),
+          "CREATE TABLE IF NOT EXISTS ks.tbl (k int, PRIMARY KEY ((k)))"),
+      arguments(
+          new QueryBuilder()
+              .create()
+              .table("ks", "tbl")
+              .ifNotExists()
+              .column("k", "int", Column.Kind.PARTITION_KEY)
+              .column("cc", "text", Column.Kind.CLUSTERING, Column.Order.DESC)
+              .column("v", "int")
+              .column("s", "int", Column.Kind.STATIC)
+              .build(),
+          "CREATE TABLE IF NOT EXISTS ks.tbl "
+              + "(k int, cc text, v int, s int STATIC, PRIMARY KEY ((k), cc)) "
+              + "WITH CLUSTERING ORDER BY (cc DESC)"),
+      arguments(
+          new QueryBuilder()
+              .create()
+              .table("ks", "tbl")
+              .column("k", "int", Column.Kind.PARTITION_KEY)
+              .withComment("'test' comment")
+              .withDefaultTTL(3600)
+              .build(),
+          "CREATE TABLE ks.tbl (k int, PRIMARY KEY ((k))) "
```

Comment:
Can we add a test for creating a table with comments? We want to make sure those work since docsapi uses them.

Reply:
Quick question on this: what does docsapi use it for? Annotating table as being used for document collection, configuration or such?

Reply:
This is a create table with a comment.
Do you mean executing the query rather than just generating it?

Reply:
Comments are used for storing a JSON schema to validate each document against.